Runtime/Integrations/Steam/SteamLobbyManager.cs

```diff
@@ -87,14 +89,14 @@ class LobbySearchBuilder : ILobbySearchBuilder {
       }
 
     public ILobbySearchBuilder Filter(string key, SearchComparison comparison, string value) {
-      var comp = (LobbyComparison)((int)comparison);
-      _query.AddStringFilter(key, value, comp);
+      //var comp = (LobbyComparison)((int)comparison);
+      //_query.AddStringFilter(key, value, comp);
```

Comment:
This actively breaks the search builder. Is this now deprecated?

Reply:
Yup it's deprecated now. I'm going to fix it to work with the new API then.